<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>621</width>
      <height>611</height>
     </rect>
    </property>
    <property name="title">
     <string>Device</string>
    </property>
    <widget class="QPushButton" name="redLed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>RED LED</string>
     </property>
    </widget>
    <widget class="QPushButton" name="blueLed">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>BLUE LED</string>
     </property>
    </widget>
    <widget class="QPushButton" name="greenLed">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>181</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>GREEN LED</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="screen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>601</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>BACK</string>
     </property>
    </widget>
    <widget class="QPushButton" name="menuButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>450</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>MENU</string>
     </property>
    </widget>
    <widget class="QPushButton" name="upButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>460</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>UP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>490</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>SELECT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>520</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>DOWN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="powerButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>570</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>POWER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="leftButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>490</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>LEFT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rightButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>490</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>RIGHT</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="batteryLevelBar">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>70</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="menuLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>601</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>MAIN MENU</string>
     </property>
    </widget>
    <widget class="QListWidget" name="mainMenuListView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>601</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="sBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>601</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>SESSION</string>
     </property>
     <widget class="QGraphicsView" name="HRVgraph">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>601</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="session_coherence">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="session_length">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="session_achievement">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="breathPacer">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>601</width>
        <height>41</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="sessionSummery">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>601</width>
       <height>381</height>
      </rect>
     </property>
     <property name="title">
      <string>SESSION SUMMERY</string>
     </property>
     <widget class="QGraphicsView" name="summery_hrvGraph">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>601</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="summeryText">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>601</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>390</y>
      <width>141</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>% BATTERY</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="batterySpinBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>45</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="chargingCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CHARGING ...</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>96</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Coherence</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>96</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Incoherence</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
